src/task-2.js

```diff
@@ -1,6 +1,10 @@
 
 export default function getNthItem(a, n) {
-  // Change me!
-
-  return 0;
+  if (n === 0) return 0;
+  const series = [];
+  series.push(a);
+  for (let i = 1; i < n; i++) {
+    series[i] = a - 2 * series[i - 1];
+  }
+  return series[n - 1];
```

Comment:
This is the correct solution. But do we need to store our sequences? I think it would be nice to rewrite this f-n without storing numbers, we do not need to store every one of them, only the last one. It will take less memory but leave the same complexity.

Reply:
I'll fix it as soon as possible)